app/models/user.rb

```diff
@@ -812,4 +812,8 @@ def initialize_attributes
   def presenter
     @presenter ||= UserPresenter.new(self)
   end
+
+  def viewable_moderation_reports
+    !is_moderator? && CurrentUser.is_moderator? ? moderation_reports : []
+  end
```

Comment:
Is this meant to say that mods can't view reports generated against other mods? Because they'll still be visible in the forum thread. Reports against mod comments would also be visible.

Allowing reports against mods to be visible to other mods is fine for now. Otherwise we'd have to get rid of the forum thread to effectively hide these.